### PR TITLE
Mvh incomplete app us17

### PR DESCRIPTION
**Pull request by mikez321**
Complete US17 with test and functionality.
Flash error message shows if required fields of application aren't filled out.
Moves flash messages into a site-wide location with standardized styling.
All controllers updated for new styling of flash messages.
All html pages have been fixed to use new location of flash message.

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -118,26 +118,42 @@ label + input {
   width: 200px;
 }
 
-.error_message {
+#error {
```
Comment (mikez321) [inline app/assets/stylesheets/application.css]: I figured these were unique types of messages so they should be IDs

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -118,26 +118,42 @@ label + input {
...
   float: left;
-  width: 50%;
+  width: 80%;
   margin-bottom: 20px;
   border-radius: 5px;
   text-align: left;
 }
 
-.well_done {
+#success {
```
Comment (elguapogordo) [inline app/assets/stylesheets/application.css]: well done!

File: app/controllers/favorites_controller.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ def create
     pet = Pet.find(params[:pet_id])
     favorite.add_pet(pet.id)
     session[:favorite] = @favorite.contents
-    flash[:pet_fav] = "Pet added to favorites."
+    flash[:success] = "Pet added to favorites."
```
Comment (mikez321) [inline app/controllers/favorites_controller.rb]: These are the new styles that can be referenced in each flash message

File: app/views/reviews/edit.html.erb [hunk 4]
```diff
@@ -1,9 +1,6 @@
 <h2>Edit Review</h2>
```
Comment (elguapogordo) [inline app/views/reviews/edit.html.erb]: <h2>Edit Review</h2>

File: app/views/reviews/new.html.erb [hunk 5]
```diff
@@ -1,8 +1,5 @@
-<h2>Add New Review</h2>
+<h1>Add New Review</h1>
```
Comment (elguapogordo) [inline app/views/reviews/new.html.erb]: <h1>Add New Review</h1>
we should keep our header tag size uniform for each CRUD page

**Pull request merged**